IO.go

```diff
@@ -64,3 +77,43 @@ func (io IO) SetDeltaTime(value float32) {
 func (io IO) SetFontGlobalScale(value float32) {
 	C.iggIoSetFontGlobalScale(io.handle, C.float(value))
 }
+
+// KeyPress sets the KeysDown flag
+func (io IO) KeyPress(key int) {
+	C.iggIoKeyPress(io.handle, C.int(key))
+}
+
+// KeyRelease clear the KeysDown flag
+func (io IO) KeyRelease(key int) {
+	C.iggIoKeyRelease(io.handle, C.int(key))
+}
+
+// KeyMap maps a key into the KeysDown array which represents your "native" keyboard state
+func (io IO) KeyMap(imguiKey int, glfwKey int) {
```

Comment:
instead of "glfwKey" it should be named "nativeKey". glfw is just one backend, it could also be SDL or others.

Reply:
note that there were a few more occurences, check for them as well, please.